src/main/scala/mesosphere/marathon/core/event/EventModule.scala

```diff
@@ -0,0 +1,93 @@
+package mesosphere.marathon.core.event
+
+import akka.actor.{ ActorRef, ActorSystem, Props }
+import akka.event.EventStream
+import akka.pattern.ask
+import mesosphere.marathon.MarathonConf
+import mesosphere.marathon.core.base.Clock
+import mesosphere.marathon.core.election.ElectionService
+import mesosphere.marathon.core.event.impl.callback._
+import mesosphere.marathon.core.event.impl.stream._
+import mesosphere.marathon.metrics.Metrics
+import mesosphere.marathon.plugin.auth.{ Authenticator, Authorizer }
+import mesosphere.marathon.state.EntityStore
+import org.eclipse.jetty.servlets.EventSourceServlet
+import org.slf4j.LoggerFactory
+
+import scala.concurrent.ExecutionContext
+
+class EventModule(
```

Comment:
Would be nice to see a little bit of documentation:
- what is the responsibility of this component
- what does this component provide